From: Li Zefan <lizf@cn.fujitsu.com>
Date: Tue, 16 Mar 2010 08:57:15 +0100
Subject: [PATCH] block: make CONFIG_BLK_CGROUP visible
Git-commit: 910ac735bad53ce54741a72a5b19ab69794ae069
References: FATE#311054
Patch-Mainline: 2.6.34

Make the config visible, so we can choose from CONFIG_BLK_CGROUP=y
and CONFIG_BLK_CGROUP=m when CONFIG_IOSCHED_CFQ=m.

Signed-off-by: Li Zefan <lizf@cn.fujitsu.com>
Signed-off-by: Jens Axboe <jens.axboe@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 block/Kconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/block/Kconfig b/block/Kconfig
index e20fbde..f9e89f4 100644
--- a/block/Kconfig
+++ b/block/Kconfig
@@ -78,8 +78,9 @@ config BLK_DEV_INTEGRITY
 	Protection.  If in doubt, say N.
 
 config BLK_CGROUP
-	bool
+	tristate "Block cgroup support"
 	depends on CGROUPS
+	depends on CFQ_GROUP_IOSCHED
 	default n
 	---help---
 	Generic block IO controller cgroup interface. This is the common
-- 
1.6.0.2

